doc/man3/EVP_desx_cbc.pod

```diff
@@ -23,6 +23,8 @@ All modes below use a key length of 128 bits and acts on blocks of 128-bits.
 
 The DES-X algorithm in CBC mode.
 
+This algorithm are available only in case when the legacy provider is loaded.
```

Comment:
_This algorithm is available only when the legacy provider is loaded._

I.e. change _are_ to _is_, and delete _in case_.